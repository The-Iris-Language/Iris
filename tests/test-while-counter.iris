.. Tests while loop with a counter
class Main() {
    public:
        int univ main() {
            int i = 0;
            while(i != 7) {
                i++;
                Olympus.print("Incremented i");
            }
            Olympus.print("while done");
        }
}