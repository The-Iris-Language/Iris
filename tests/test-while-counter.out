Incremented i
Incremented i
Incremented i
Incremented i
Incremented i
Incremented i
Incremented i
while done
